### ISSUE TITLE
[MSFT-112] Member registration page: The help content related to the 'email' 'user name' and 'password' fields is not announced when user navigates with tab key.

### ISSUE THREAD

**Issue by veshu** [importtest/ewe#76]
**Steps to Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 2. Ensure that the screen reader is switched on. 3. Press tab till your arrive at the 'enter your email id' text box. 4. Notice that the screen reader doesn't announce the help content. 5. Press tab to arrive at the 'user name' and 'password' text fields and observe that the help content is skipped.

**Expected Result**
The help content should be announced by screen readers when user navigates through the form fields with tab key. Since the help content is not tagged to the respective fields, the announcement is not made.

**Actual Result**
Screen reader doesn't announce the help content associated with the 'email' 'user name' and 'password' fields when user navigates with tab key alone. There is no indication to the user as to the presence of such help content unless and until he/she navigates with arrow keys.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2017
